galloverse_ics3u_b.py

```diff
@@ -729,7 +752,33 @@ def linear_interpolation(x, x0, x1, y0, y1):
 
 
     # ----------------------------------------------------------------------------------------
+running = True
+while running:
+    for event in pygame.event.get():
+        if event.type == pygame.QUIT:
+            running = False
+
+    # Clear the screen with sky background
+    screen.fill(sky_color)
+
+    # Update the sun position
+    sun_x -= sun_speed
```

Comment:
Animation variables like `sun_x` need to be named with your last name to avoid conflicts.